pkg/operator/ceph/config/network.go

```diff
@@ -50,10 +50,8 @@ func generateNetworkSettings(clusterdContext *clusterd.Context, namespace string
 	cephNetworks := []Option{}
 
 	for _, selectorKey := range NetworkSelectors {
-		// This means only "public" was specified and thus we use the same subnet for cluster too
+		// This means only one network selector was specified
 		if _, ok := networkSelectors[selectorKey]; !ok {
-			cephNetworks = append(cephNetworks, cephNetworks[0])
-			cephNetworks[1].Option = fmt.Sprintf("%s_network", selectorKey)
 			continue
 		}
```

Comment:
It's not clear to me how this means only one network selector is specified? What is "This" referring to, and how do we know only one is specified in this loop?